### PR TITLE
fix(issue_templates) change wording and fix grammatical issues

### PR DESCRIPTION
**Pull request by ArjunSharda**
Changes wording of some issue templates and fixes some grammatical issues I have spotted.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "📚 Read docs"
+  - name: "📚 View the docs"
```
Comment (JoanFM) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: "📚 Read the docs"
```

File: .github/ISSUE_TEMPLATE/---can-jina-do-this-.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: "\U0001F64B Can Jina do this?"
-about: Open question such as: I want to use Jina for X, but not sure is doable.
+about: Open question such as: I want to use Jina for X, but not sure if Y is doable.
```
Comment (JoanFM) [inline .github/ISSUE_TEMPLATE/---can-jina-do-this-.md:3]:
```suggestion
about: Open question such as: I want to use Jina for X, but not sure if it is doable.
```

**Review by JoanFM — CHANGES_REQUESTED** [review]
Thanks for contributiing, but I think there are some changes to be applied

**Comment by ArjunSharda** [comment]
Added

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "📚 Read docs"
+  - name: "📚 Read the docs"
     url: https://docs.jina.ai/
-    about: Find your solution from our documenations
+    about: Find your solution from our documentations
```
Comment (JohannesMessner) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Find your solution from our documentation
```

**Comment by ArjunSharda** [comment]
> Hey @ArjunSharda ,
> 
> 
> 
> Thanks for the contribution, can you please check our `https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md` guidelines to check how to have a proper commit message?
Updated.

**Comment by JoanFM** [comment]
The best way is to open a new PR from a new branch with a correct commit message

**Comment by ArjunSharda** [comment]
> > The best way is to open a new PR from a new branch with a correct commit message
> 
> You mean like the commit message descriptions?

Bumping this, @JoanFM

**Pull request closed without merging**